app/controllers/api/base_controller/parameters.rb

```diff
@@ -82,6 +82,9 @@ def parse_filter(filter, operators)
                         [filter_value.gsub(/^'|'$/, ''), str_method]
                       when /^".*"$/
                         [filter_value.gsub(/^"|"$/, ''), str_method]
+                      when /^\d{4}-\d{2}-\d{2}/
```

Comment:
@abellotti even if we do detect well-formed dates and times, it will still 500 if the user gives some bogus value for a date or datetime field - problem?